### ISSUE TITLE
Feature Request: Activitiy-List over all Boards

### ISSUE THREAD

**Issue by hingerl** [wekan/wekan#1377]
please add an Activity-List over all Boards, for better overview.

thanks.

**Comment by frank-runge** [comment]
Yes, for working in a team in my opinion this kind of notification is needed: If I don't realize that someone has done or left something, then the whole tool becomes quite blunt.

_trello has the little bell in the upper right corner: here all activities (concerning me, similar to "tracking" at Wekan) are listed permanently and across all boards. And if there is a notification under the bell that I haven't read yet, then the bell is highlighted._

**Comment by d3dbit** [comment]
It could be a great improvement:
"*trello has the little bell in the upper right corner: here all activities (concerning me, similar to "tracking" at Wekan) are listed permanently and across all boards. And if there is a notification under the bell that I haven't read yet, then the bell is highlighted.*"
